Assets/Modules/Obstacles/Scripts/ObstacleEntity.cs

```diff
@@ -4,7 +4,8 @@
 
 public class ObstacleEntity : MonoBehaviour
 {
-    [SerializeField] bool hooknessActivated;
+    [SerializeField] public bool hooknessActivated;
```

Comment:
one detail, when you declare a variable as `public` in a MonoBehaviour you don't neet the [Serializefield] attribute anymore. Always use [Serializefield] in combination with `private`. It doesn't change anything but is just more logical.